module/variable.go

```diff
@@ -0,0 +1,18 @@
+package module
+
+import (
+	"github.com/hashicorp/hcl-lang/schema"
+	"github.com/hashicorp/hcl/v2"
+	"github.com/zclconf/go-cty/cty"
+)
+
+type Variable struct {
```

Comment:
Do you mind adding `Variables` map leveraging this to `module.Meta`?

Reply:
Got it makes more sense to put that there them instead of where I was adding that directly into terraform-ls

Reply:
We'll need them in both places actually - which may look like a duplication, but this helps us avoid any unnecessary dependencies between the two.